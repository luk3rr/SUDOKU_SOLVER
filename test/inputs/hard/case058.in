000060000 083500010 400032070 000050020 000000709 000306040 600029007 050000200 370000008 
