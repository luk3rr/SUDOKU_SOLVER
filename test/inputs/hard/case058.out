Solving the following grid:
0 0 0 | 0 6 0 | 0 0 0 
0 8 3 | 5 0 0 | 0 1 0 
4 0 0 | 0 3 2 | 0 7 0 
------+-------+------
0 0 0 | 0 5 0 | 0 2 0 
0 0 0 | 0 0 0 | 7 0 9 
0 0 0 | 3 0 6 | 0 4 0 
------+-------+------
6 0 0 | 0 2 9 | 0 0 7 
0 5 0 | 0 0 0 | 2 0 0 
3 7 0 | 0 0 0 | 0 0 8 


Solution found :')

5 9 7 | 4 6 1 | 3 8 2 
2 8 3 | 5 9 7 | 4 1 6 
4 6 1 | 8 3 2 | 9 7 5 
------+-------+------
7 1 4 | 9 5 8 | 6 2 3 
8 3 6 | 2 1 4 | 7 5 9 
9 2 5 | 3 7 6 | 8 4 1 
------+-------+------
6 4 8 | 1 2 9 | 5 3 7 
1 5 9 | 7 8 3 | 2 6 4 
3 7 2 | 6 4 5 | 1 9 8 

Algorithm: BFS
Total time: 1044 ms
Total expanded states: 163959
